src/DotNetWorker.ApplicationInsights/FunctionsApplicationInsightsExtensions.cs

```diff
@@ -44,11 +43,13 @@ public static IFunctionsWorkerApplicationBuilder AddApplicationInsightsLogger(th
         {
             builder.AddCommonServices();
 
+            // Lets the host know that the worker is sending logs to App Insights. The host will now ignore these.
+            builder.Services.Configure<WorkerOptions>(workerOptions => workerOptions.Capabilities["WorkerApplicationInsightsLoggingEnabled"] = bool.TrueString);
```

Comment:
Would this work fine with the changes exposed in #1183 where we allow users to override/disable default values of capability ?

Reply:
Yes, that's why I introduced that in the first place. B/c these are `Options<>`, you can register your own `ConfigureOptions<>` later to remove this if you need to for some reason. They'll chain together.